fmriprep/workflows/base.py

```diff
@@ -439,20 +439,20 @@ def init_single_subject_wf(subject_id):
         if estimator.method in (fm.EstimatorType.MAPPED, fm.EstimatorType.PHASEDIFF):
             continue
 
-        suffices = set(s.suffix for s in estimator.sources)
-
-        if estimator.method == fm.EstimatorType.PEPOLAR and sorted(suffices) == ["epi"]:
-            getattr(fmap_wf.inputs, f"in_{estimator.bids_id}").in_data = [
-                str(s.path) for s in estimator.sources
-            ]
-            getattr(fmap_wf.inputs, f"in_{estimator.bids_id}").metadata = [
-                s.metadata for s in estimator.sources
-            ]
-
-        elif estimator.method == fm.EstimatorType.PEPOLAR:
-            raise NotImplementedError(
-                "Sophisticated PEPOLAR schemes are unsupported."
-            )
+        suffices = [s.suffix for s in estimator.sources]
```

Comment:
just so i'm clear - this was changed to ensure there is only 1 bold / fmap pair in the estimation?

Reply:
Yes, that or all EPI, which we previously allowed.